podvm/Makefile.inc

```diff
@@ -26,10 +38,10 @@ COMMIT := $(shell	commit=$$(git describe --match '' --dirty --always) && \
 ifndef COMMIT
 $(error Failed to derive an image name. Explicitly define IMAGE_NAME)
 endif
-IMAGE_NAME := $(IMAGE_PREFIX)-$(COMMIT)-$(ARCH)
+IMAGE_NAME := $(IMAGE_PREFIX)-$(COMMIT)-$(DEB_ARCH)
 endif
 
-IMAGE_SUFFIX ?= 
+IMAGE_SUFFIX ?=
 IMAGE_FILE = $(IMAGE_NAME)$(IMAGE_SUFFIX)
 
 LIBC_ARCH := $(shell uname -m)
```

Comment:
We should be able to remove these lines as `LIBC` is now defined above and uses `ARCH` instead.